src/ucp/wireup/wireup_cm.c

```diff
@@ -1345,6 +1345,10 @@ void ucp_ep_cm_disconnect_cm_lane(ucp_ep_h ucp_ep)
     ucp_ep_update_flags(ucp_ep, UCP_EP_FLAG_DISCONNECTED_CM_LANE,
                         UCP_EP_FLAG_LOCAL_CONNECTED);
 
+    /* Remove all CM-related callbacks from the callbackq to make sure they
+     * won't be invoked */
+    ucp_ep_cm_slow_cbq_cleanup(ucp_ep);
```

Comment:
1. should we do it in ucp_ep_remove_progress_callbacks?
2. does it fix a test failure?

Reply:
`ucp_ep_remove_progress_callbacks()` - it is called when destroying EP.
but disconnect CM lane can be done even if EP is not going to be destroyed right now - e.g. handle failure.
`ucp_ep_cm_slow_cbq_cleanup()` is called also when destroying EP

Reply:
> does it fix a test failure?

yes, I though the problem in CI, but can't find the backtrace